tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,33 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+import time
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_app = {
+            'name': 'Bugzilla Lite Stage',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_app)
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % self.test_app['url'])
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
+        self.push_resource('IMG_0001.jpg')
```

Comment:
yes they have the same methods, I don't think it's a good idea to put it before  connect_to_local_area_network() because we need network connected to install the app no ? and as I know the bugzilla lite doesn't work without connection